### DISCUSSION TITLE
Tracking Multi-Animal Accurately

### DISCUSSION THREAD

**Discussion by djg9324** [talmolab/sleap#1609] [Help!]
I have been trying to train a model to track two mice in an arena. I currently have 350 frames and am working on predicting another 50 at the moment. As I train and run inference, I notice that the model has not improved in differentiating the two mice. Once mouse has a shaved spot in its back while the other does not, so I am able to tell which mouse is which; though the computer seems to have trouble doing that. Are there any parameters that I should be looking into intensely to improve the model's training?
<img width="1043" alt="Screenshot 2023-11-27 at 12 19 34 PM" src="https://github.com/talmolab/sleap/assets/139586492/92adfb1a-22ca-43af-9038-fddba3831f2e">

**Comment by talmo** [comment]
Hi @djg9324,

Apologies for the delay -- do you mind sharing a screenshot of the model configuration that you're training, or clicking the "copy to clipboard" button in the Run Training window? It'll help us diagnose what's going on :)

If you have more screenshots of examples of when things go wrong that'll help too.

Thanks!

Talmo
